spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/DefaultAccessTokenConverter.java

```diff
@@ -73,6 +91,15 @@ public void setScopeAttribute(String scopeAttribute) {
 		this.scopeAttribute = scopeAttribute;
 	}
 
+	/**
+	 * Scope attribute name to be used in the converted token.
+	 *
+	 * @return the scope attribute name to use
+	 */
+	public String getScopeAttribute() {
```

Comment:
This does not need to be exposed. It's only required internally for `convertAccessToken` and `extractAuthentication`.